src/Property/Controller/Element/CollectionController.php

```diff
@@ -45,7 +46,7 @@ public function __construct(
     }
 
     /**
-     * @throws NotFoundException
+     * @throws NotFoundException|AccessDeniedException
      */
     #[Route('/properties/{elementType}/{id}', name: 'pimcore_studio_api_get_element_properties', methods: ['GET'])]
     //#[IsGranted('STUDIO_API')]
```

Comment:
Should we remove this comment? Or is it still relevant for the future?

Reply:
No we can remove it completely :)